Src/FluentAssertions/Types/TypeSelector.cs

```diff
@@ -183,6 +183,24 @@ public TypeSelector ThatAreNotClasses()
         return this;
     }
 
+    /// <summary>
+    /// Determines whether the type is abstract
```

Comment:
🔧 It doesn't determine whether a type is abstract. It filters them. Same with the other methods.